lib/KTable/index.vue

```diff
@@ -718,24 +856,46 @@
   td {
     position: relative;
     z-index: auto;
-    padding: 8px;
+    padding: 0 8px;
   }
 
   .sticky-header {
     position: sticky;
     top: 0;
-    z-index: 2;
+    z-index: 10;
```

Comment:
It looks like the increased z-index values are causing the headers and columns to be displayed over the AppBar in Kolibri, because the [ScrollingHeader](https://github.com/learningequality/kolibri/blob/6b54f83bade08d2a989df9f525457422b59ccba1/packages/kolibri/components/pages/ScrollingHeader.vue#L166) has a z-index of 4.

![Screenshot 2025-06-26](https://github.com/user-attachments/assets/c21ba994-dde4-4978-913d-31c2e3267726)